gradlew.bat

```diff
@@ -5,7 +5,7 @@
 @rem you may not use this file except in compliance with the License.

```

Comment:
It is better to leave out the generated Gradle tools unless the Gradle version has changed.